charts/miggo-operator/templates/_helpers.tpl

```diff
@@ -154,15 +154,13 @@ The image to use for opentelemetry-operator.
 {{- define "imagePullSecrets" -}}
 {{- if (empty (index .Values.imagePullSecrets 0).name) }}
 imagePullSecrets:
-  - name: regcred
+  - name: miggo-regcred
```

Comment:
it wont break nothing in the operator?  it was hard coded there but don't remember if you fixed it